### PR TITLE
fix: matomo queue 0 handling

### PR DESCRIPTION
**Pull request by alexgarel**

### CODE REVIEW

**Comment by alexgarel** [comment]
@odtvince what I did today !

File: docs/reports/2024-02-13-matomo-fix-queue0.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+systemctl start matomo-tracking@0.timer
+
+# start a screen
+screen -S matomo-queue
+
+# start a process to consume the renamed list, starting again on errors
+while true; do /var/www/html/matomo/console queuedtracking:process --queue-id=4 -v -n;/var/www/html/matomo/console queuedtracking:lock-status --unlock=QueuedTrackingLock4 -n --ignore-warn; sleep 60; done
+```
+
+Monitoring progress
```
Comment (stephanegigandet) [inline docs/reports/2024-02-13-matomo-fix-queue0.md:40]: What command do you use to see the number of items in each queue?

**Review by stephanegigandet — APPROVED** [review]
Thanks

File: docs/reports/2024-02-13-matomo-fix-queue0.md (hunk 1 above)
Comment (alexgarel) [reply docs/reports/2024-02-13-matomo-fix-queue0.md:40]: I will add it ! (very simple)

**Pull request merged**